cli/command/image/import.go

```diff
@@ -88,3 +88,7 @@ func runImport(dockerCli command.Cli, options importOptions) error {
 
 	return jsonmessage.DisplayJSONMessagesToStream(responseBody, dockerCli.Out(), nil)
 }
+
+func isURL(str string) bool {
+	return strings.HasPrefix(str, "https://") || strings.HasPrefix(str, "http://")
```

Comment:
why not use https://github.com/moby/moby/blob/master/builder/remotecontext/urlutil/urlutil.go#L20 (as recommended by deprecation notice)? This introduces code duplication
Also method name sounds weird to me, as there's more than just http to be actual URLs

Reply:
I moved it to `builder/remotecontext` as that utility should really only be used for that purpose, and not as a generic utility to check "if it's a URL", as `docker build` may have different "rules" than other users.

Reply:
I removed the local `isURL()` function (to not give the illusion we're validating it's an actual URL), and changed the logic to use a switch, to make it slightly more clearer what the criteria are for each.